components/withBlogCrossLinks.tsx

```diff
@@ -22,7 +22,7 @@ const WithBlogCrossLinks: FC = async () => {
   ];
 
   return (
-    <div className="mt-4 grid w-full grid-cols-2 gap-4 xs:grid-cols-1">
+    <div className="mt-4 grid w-full grid-cols-1 gap-4 sm:grid-cols-2">
```

Comment:
why theses change.  did you use the figma or you made the changes by eye.

Reply:
Yes, I referred figma for the responsive changes. The changes follow the design to the dot. I changed it because xs uses max-width 670px and it was off by 30px with the other changes in the layout. It is just a minor detail. Can revert it if it's not ok.

Reply:
Ok that good 👍🏼  !